content/influxdb/v2.1/tools/chronograf.md

```diff
@@ -20,16 +20,15 @@ that provides an InfluxQL data explorer, Kapacitor integrations, and more.
 Continue to use Chronograf with **InfluxDB Cloud** and **InfluxDB OSS {{< current-version >}}** and the
 [1.x compatibility API](/influxdb/v2.1/reference/api/influxdb-1x/).
 
+
 ## Create an InfluxDB connection
-1. In Choronograf, click **Configuration** in the left navigation bar,
```

Comment:
"Choronograf" 🤣🤣🤣 I'm pretty sure I did this. Checking the git blame.... yup, it was me 🤣